schema.graphql

```diff
@@ -110,3 +105,17 @@ type UnprocessedEntity @entity {
   transaction: Transaction
   block: Block
 }
+
+type BalanceOfAccountByDenom @entity {
+  id: ID!
+  account: Account! @index
+  denom: String! @index
+  amount: BigInt!
+  lastUpdatedBlock: Block!
+}
```

Comment:
:ok_hand: Nice!

Reply:
What do you think about simplifying the name to something like `Balance` or `CurrentBalance`? I feel like it's conventional that balances will be denom specific.